frontends/mit-open/src/page-components/TestimonialDisplay/TestimonialDisplay.tsx

```diff
@@ -0,0 +1,197 @@
+import React from "react"
+
+import { ActionButton, styled, theme } from "ol-components"
+import { useTestimonialList } from "api/hooks/testimonials"
+import { Attestation } from "api/v0"
```

Comment:
```suggestion
import type { Attestation } from "api/v0"
```
IMO `type` imports are nice when you only need the type. The are 100% guaranteed to have no impact at runtime (whereas `import { Attestation } from "api/v0"` can have an impact even if you're only importing a type, if `api/v0` has side effects.)

Also makes intent clear.